tests/integration/pilot/vm/testdata/OS_Support_List

```diff
@@ -0,0 +1,6 @@
+app_sidecar_trusty
+app_sidecar_xenial
+app_sidecar_focal
+app_sidecar_debian8
+app_sidecar_debian9
+app_sidecar_debian10
```

Comment:
Ensure that you're ending all files with newlines. If you are using an IDE you can auto add newlines at end of all files.

Reply:
Thanks! I will add a new line.